spec/features/open_course_creation_spec.rb

```diff
@@ -42,6 +42,7 @@ def fill_out_open_course_creator_form
   it 'lets a user create a course immediately', js: true do
     visit root_path
     click_link 'Create an Independent Program'
+    all('.program-description').each &:click
```

Comment:
Is the idea here that you click each program type in order, and it will end up as the last one clicked?

Reply:
Yes, that's what i am trying to do

Reply:
This was passing even without the change that made it not skip the type chooser, which means it's not a precise enough test to indicate whether the chooser is doing what we want: `.each` will work find even if the set is empty (as it was previously).

I think instead this should click one option (and then it could explicitly click a different one), and at the end, it should have a check to see that the correct course type got set.

Reply:
okay